classes/core-image-pelux.bbclass

```diff
@@ -28,6 +28,7 @@ IMAGE_INSTALL_append = "\
 
 # Alexa SDK
 IMAGE_INSTALL_append = "\
+    portaudio-v19      \
```

Comment:
am I missing something? why it's not simply a dependency of the sdk?

Reply:
Since the portaudio has shared library. It has to be added to the image so that they are shipped with the image.

Reply:
why?

Reply:
Because It is being used by the sample application.

Reply:
I am not sure but probably it will not be needed to explicitly add to the image if portaudio is set as an rdepends instead of depends. May be then it will implicitly add it to the image. I will try that and if it works then its not needed here. Will come back.

Reply:
I have tested now. So adding portaudio as rdepends will implicitly add it to the final image. Closing this PR.